### DISCUSSION TITLE
Should Unique Visitors or Total Pageviews be used as a metric for how many times an article has been read?

### DISCUSSION THREAD

**Discussion by lloydjatkinson** [plausible/analytics#2435] [Anything Else]
Hi. Which statistic should be used as a way of getting the article read count? I think Total Pageviews but wanted to double check I hadn't missed anything obvious. I see such a large gap between the two stats I wasn't sure which one to use for that purpose.

![image](https://user-images.githubusercontent.com/5633938/201522313-0941a86b-92fa-401a-bd91-c77e8eba9659.png)

**Comment by metmarkosaric** [comment]
hi @lloydjatkinson! whichever you prefer really. unique visitors will show you the number of unique people who have read the article while the total pageviews will show you how many times the people have viewed the article in total. in your screenshot, you've had 2.5 page views for every unique visitor

**Comment by lloydjatkinson** [comment]
Thank you. Looking at other stats I'm still not sure I can believe the difference, for example are there really 4k people all viewing the same page almost 3 times instead of just once?

![image](https://user-images.githubusercontent.com/5633938/204175007-f527f13c-7565-4b90-a260-aacc88a7db11.png)

**Reply by metmarkosaric** [reply]
you're welcome! doesn't necessarily have to be all of them. some might view it once, some might view it multiple times. it really depends on the content and the quality of traffic. you seem to have a really low bounce rate which is unusual but as long as you've installed Plausible correctly, it just means that you have a sticky site that your visitors find engaging.
